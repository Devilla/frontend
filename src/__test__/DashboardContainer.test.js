// import React from 'react';
// import { shallow, mount } from 'enzyme';
// import renderer from 'react-test-renderer'
// import configureStore from 'redux-mock-store'
// import { Provider } from 'react-redux'
// import {createStore} from 'redux'
// import { load, loaded } from 'ducks/loading';
// import { DashboardContainer } from 'containers';
// import { store } from 'App';
//
// // Snapshot for Home React Component
// describe('>>> DashboardContainer --- Snapshot',()=>{
//     it('+++capturing Snapshot of DashboardContainer', () => {
//         const renderedValue =  renderer.create(<DashboardContainer store={store} />).toJSON()
//         expect(renderedValue).toMatchSnapshot();
//     });
// });
