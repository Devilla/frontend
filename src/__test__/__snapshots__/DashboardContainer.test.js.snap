// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> DashboardContainer --- Snapshot +++capturing Snapshot of DashboardContainer 1`] = `
<div
  className="dashboard-container"
>
  <div
    className="mm-popup"
  >
    <div
      className="mm-popup__overlay"
      onClick={[Function]}
      role="presentation"
      style={Object {}}
    />
  </div>
  <div
    className="wrapper"
  >
    <div
      className="left side-menu"
    >
      <div>
        <div
          className="user-box"
        >
          <h5 />
          <p
            className="text-muted"
          />
        </div>
        <div
          className="topbar-left mt-3 ml-2 pt-2 pl-2"
          style={
            Object {
              "width": "200px",
            }
          }
        >
          <a
            className="logo "
            onClick={[Function]}
            style={Object {}}
          >
            <span>
              <img
                alt="influence-img"
                className="ml-2"
                height="40"
                src="newlogo-6.png"
              />
            </span>
          </a>
        </div>
        <div
          id="sidebar-menu"
        >
          <div
            className="button-list ml-2 pl-2"
          >
            <a
              onClick={[Function]}
              style={Object {}}
            >
              <button
                className="btn btn-primary waves-effect  addnew-btn  ml-4 p-2  pt-0 pb-0  w-lg "
                type="button"
              >
                <i
                  className="fi-plus "
                />
                Â 
                 
                <span
                  className="h6"
                >
                  New
                </span>
                 
              </button>
            </a>
          </div>
          <ul
            className="metismenu mt-5"
            id="side-menu"
          >
            <li
              className="/dashboard"
            >
              <a
                className="nav-link"
                disabled={false}
                onClick={[Function]}
                style={Object {}}
              >
                <i
                  className="fi-air-play"
                />
                <span>
                  Home
                </span>
              </a>
            </li>
            <li
              className="/campaigns"
            >
              <a
                className="nav-link"
                disabled={false}
                onClick={[Function]}
                style={Object {}}
              >
                <i
                  className="fi-layers"
                />
                <span>
                  Campaigns
                </span>
              </a>
            </li>
            <li
              className="/analytics"
            >
              <a
                className="nav-link"
                disabled={false}
                onClick={[Function]}
                style={Object {}}
              >
                <i
                  className="fi-bar-graph-2"
                />
                <span>
                  Analytics
                </span>
              </a>
            </li>
            <li
              className="/integrations"
            >
              <a
                className="nav-link"
                disabled={false}
                onClick={[Function]}
                style={Object {}}
              >
                <i
                  className="fi-command"
                />
                <span>
                  Integrations
                </span>
              </a>
            </li>
            <div
              className="custombottom ml-2 mb-5"
            >
              <hr />
              <div
                className="pt-4 col-md-12"
              >
                <div
                  className="text-center"
                >
                  <div
                    className="progress"
                  >
                    <div
                      aria-valuemax={100}
                      aria-valuemin={0}
                      aria-valuenow={0}
                      className="progress-bar progress-bar-info active progress-bar-striped"
                      role="progressbar"
                      style={
                        Object {
                          "width": "0%",
                        }
                      }
                    />
                  </div>
                </div>
                <div
                  className="ml-4 "
                >
                  <p
                    className="textColor"
                  >
                    0
                     % consumed.
                  </p>
                </div>
                <hr />
              </div>
            </div>
          </ul>
        </div>
        <div
          className="clearfix"
        />
      </div>
    </div>
    <div>
      <div
        className="content-page"
      >
        <div
          className="topbar"
        >
          <nav
            className="navbar-custom pl-0 pr-0"
          >
            <div
              className="customer-header"
            >
              <div
                className="progress"
              >
                <div
                  aria-valuemax={100}
                  aria-valuemin={0}
                  aria-valuenow={0}
                  className="progress-bar progress-bar-info"
                  role="progressbar"
                  style={
                    Object {
                      "width": "0%",
                    }
                  }
                />
              </div>
              <div />
              <div
                className="nav-topbar-flex"
              >
                <div
                  className="topbar-left"
                >
                  <h4>
                    <a
                      onClick={[Function]}
                      style={Object {}}
                    >
                      <i
                        className="icon-arrow-left mr-3"
                      />
                    </a>
                  </h4>
                </div>
                <ul
                  className="list-unstyled topbar-right-menu float-right mb-0 nav-custom-header  rounded-circle"
                >
                  <li
                    className="dropdown notification-list"
                  >
                    <a
                      aria-expanded="false"
                      aria-haspopup="false"
                      className="nav-link dropdown-toggle nav-user"
                      data-toggle="dropdown"
                      onClick={[Function]}
                      role="button"
                      style={
                        Object {
                          "cursor": "pointer",
                        }
                      }
                    >
                      <i
                        className=" mdi mdi-account-settings-variant"
                      />
                      <span
                        className="ml-1"
                      />
                    </a>
                    <div
                      className="dropdown-menu dropdown-menu-right dropdown-menu-animated profile-dropdown"
                      style={Object {}}
                    >
                      <div
                        className="dropdown-item noti-title"
                      >
                        <h6
                          className="text-overflow m-0"
                        />
                      </div>
                      <a
                        className="dropdown-item notify-item"
                        href="javascript:void(0);"
                      >
                        <i
                          className="fi-head"
                        />
                        <span
                          onClick={[Function]}
                        >
                          Your Profile
                        </span>
                      </a>
                      <a
                        className="dropdown-item notify-item"
                        href="javascript:void(0);"
                        onClick={[Function]}
                      >
                        <i
                          className="fi-help"
                        />
                        <span>
                          Support
                        </span>
                      </a>
                      <a
                        className="dropdown-item notify-item"
                        href="javascript:void(0);"
                        onClick={[Function]}
                      >
                        <i
                          className="fi-power"
                        />
                        <span>
                          Logout
                        </span>
                      </a>
                    </div>
                  </li>
                </ul>
              </div>
            </div>
          </nav>
        </div>
        <div
          className="content dashboard-content"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#FFF",
            }
          }
        >
          <div
            className="container-fluid p-2"
          />
        </div>
      </div>
    </div>
  </div>
  <div />
</div>
`;
